modules/accredited_representative_portal/app/services/accredited_representative_portal/rep_user_loader.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module AccreditedRepresentativePortal
+  class RepUserLoader < SignIn::UserLoader
```

Comment:
Would it be difficult to copy the `UserLoader` functions into here and remove the inheriting? Or we can unravel it with a followup PR

Reply:
I can copy them over